phase1/gce/do

```diff
@@ -7,7 +7,14 @@ set -x
 
 cd "${BASH_SOURCE%/*}"
 
-CLUSTER_NAME=$(jq -r '.phase1.cluster_name' ../../.config.json)
+if [[ "${2:-}" == "" ]]; then
```

Comment:
This might be bikeshedding, but passing the config file in as a second optional argument feels a little weird here. It would be different if we had proper argument parsing in these scripts and added an optional `--config <file>` option, but the way we have generally been passing around optional information is through environment variables. Especially given that you're proposing to override the default .config file via an environment variable, what do you think about using the same convention here?

Another bonus is that you can export this environment variable once in the Makefile and it will automatically propagate to the child processes, so you don't have to explicitly plumb it through to the `do` steps.